object-manipulation-1/src/get-keys.ts

```diff
@@ -0,0 +1,10 @@
+/* exported getKeys */
+function getKeys(object: Record<string, any>): string[] {
+  const keys: string[] = [];
+  for (const key in object) {
+    if (key in object) {
+      keys.push(key);
+    }
+  }
```

Comment:
This check isn't necessary. Your `for...in` loop is going through every `key` in your object and then you are checking if the `key` exists in the object but you wouldn't enter the loop if the `key` did not exist.